GitLab Pipeline Status Test Failing

:clock11: **When did the problem start?**
~October 30th

<!-- Indicate when the problem started -->

:camera: **Live badge**
https://img.shields.io/gitlab/pipeline/gitlab-org/gitlab-ce
<!-- Provide a link to the live badge in plain text and markdown.  -->

![](https://img.shields.io/gitlab/pipeline/gitlab-org/gitlab-ce)

:wrench: **Is the live badge working?**
Yup
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://circleci.com/gh/badges/daily-tests/696
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
1. GitlabPipelineStatus [live] Pipeline status (nonexistent repo) [ GET /this-repo/does-not-exist.json ]
  AssertionError: message mismatch: expected 'not found' to equal 'repo not found'
```

:bulb: **Possible solution**
It seems like this test is failing because the upstream endpoint is no longer returning a 401 response code. I'm seeing 200 responses with a login page for a non-existent repo. 

A simple fix would be to update the test expectation from _repo not found_ to _not found_ 

<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/gitlab/gitlab-pipeline-status.service.js
+++ b/services/gitlab/gitlab-pipeline-status.service.js
@@ -91,6 +91,7 @@ module.exports = class GitlabPipelineStatus extends BaseSvgScrapingService {
       url: `${baseUrl}/${user}/${repo}/badges/${branch}/pipeline.svg`,
       errorMessages: {
         401: 'repo not found',
+        404: 'repo not found',
       },
     })
     if (status === 'unknown') {

